### PR TITLE
fixed screen reader not reading view with i4a='Yes' in RS4 or <=RS3+N…

### PR DESCRIPTION
**Pull request by zholobov**
…VDA. this change also removes suffix "group" when calling announceForAccessibility

### CODE REVIEW

**Comment by zholobov** [comment]
@rozele could you take a look? this is an important accessibility fix. @gmangov already reviewed this.

**Comment by rozele** [comment]
@zholobov in what case are these methods called for a `UIElement` that is not from React Native?

**Comment by zholobov** [comment]
@rozele 
>in what case are these methods called for a UIElement that is not from React Native?

For example for the Border inside BorderCanvas. For UIElements that are children of RichTextBlock. Is there a problem in the case when it's not a React view?

>How does UWP in general handle screen reading for a view that is actively being removed? Is this a problem with the lifecycle management of the view? E.g., should we instead only remove some properties when the view is fully unloaded?

The case where it crashed was when a React view was removed from tree, but reference to the native view was held transitively by screen reader. Something like: BordedCanvas <- DynamicAutomationPeer <- .NET/XAML UIA implementation <- OS UIA infra <- A screen reader. One specific example is when on splash screen app calls RX.Accessibility.announceForAccessibility, this gets to raising LiveRegionChanged by RNW, this finally queues announcement in a screen reader and this announcement holds refcounted COM reference that finally holds native managed view instance in .NET and when reader gets to announce it does all kinds of calls to the peer (which calls the view) to determine text to announce and if announcement still needs to happen.

**Comment by rozele** [comment]
@zholobov - is this something that can occur in a typical UWP scenario? I.e., cases where a view is removed but the screen reader still has a pointer to it? Or is this unique to react-native-windows?

File: ReactWindows/ReactNative/Accessibility/AccessibilityHelper.cs [hunk 1]
```diff
@@ -292,6 +309,23 @@ public static void SetAccessibilityLabel(UIElement element, string accessibility
...
         }
 
+        /// <summary>
+        /// Gets the AccessibilityLabel property for <paramref name="element"/>.
+        /// </summary>
+        /// <param name="element"></param>
+        /// <returns>Null if the <paramref name="element"/> is not a React Native view, value of the property otherwise.</returns>
+        public static string GetAccessibilityLabel(UIElement element)
+        {
+            if (element.HasTag())
```
Comment (rozele) [inline ReactWindows/ReactNative/Accessibility/AccessibilityHelper.cs]: Can simplify a bit with:
```
return element.HasTag() ? GetAccessibilityLabelProp(element) : null;
```

**Comment by zholobov** [comment]
>is this something that can occur in a typical UWP scenario? I.e., cases where a view is removed but the screen reader still has a pointer to it? Or is this unique to react-native-windows?

Yes, it's usual XAML scenario. `AutomationPeer`s should be very careful because they are invoked from OS directly at random times.

File: ReactWindows/ReactNative/Accessibility/AccessibilityHelper.cs (hunk 1 above)
Comment (zholobov) [reply ReactWindows/ReactNative/Accessibility/AccessibilityHelper.cs]: Done. See third commit.

**Comment by zholobov** [comment]
@rozele Sorry for compilation error. It's fixed now. Could you review this?

**Pull request merged**